### ISSUE TITLE
Regarding tomcat service

### ISSUE THREAD

**Issue by manojhiox** [vivex/cPanel4j#1]
I have installed your tomcat script on my cpanel server.But i found that each users tomcat is running on root only.How to start and start service for each user in SSH and one more thing ajp is not properly configure in proxy.

**Comment by manojhiox** [comment]
First of all applause for your service.

**Comment by vivekfyi** [comment]
Yeah each user's tomcat instance is running on root , if you want that it should run by user only, then you need to make main tomcat's bin/ dir to be executable by all users. Would that be safe?

What configuration issue you found in ajp config, right now we are adding these two lines to user's domain's vhost file - 

ProxyPass / ajp://localhost:8181/ 
ProxyPassReverse / ajp://localhost:8181;
8181 is ajp port of that user.
